Dockerfile

```diff
@@ -27,4 +27,3 @@ COPY /etc/nginx/nginx.conf /etc/nginx/nginx.conf
 COPY /etc/nginx/conf.d/default-server.conf /etc/nginx/conf.d/default-server.conf
 
 VOLUME /var/log/nginx
-EXPOSE 80
```

Comment:
I'm removing port 80 because the base container we are using (nginx) exposes port 80 and 443 by default. So this was an unnecessary entry.